wiki/configuration/advanced-configuring/en.md

```diff
@@ -0,0 +1,270 @@
+Now instead of dealing with `ExampleModConfig.INSTANCE.advancedSettings.printStream.value().getStream()` we can simply do `ExampleModConfig.INSTANCE.advancedSettings.activeStream`, simplifying our lives a little when interacting with the config. The power of processors, in action.
+
+## Changing the config format
+
+Let's get into how you choose a file format to save to. Quilt Config currently only provides two serializers: `json5`, an extension of the JSON format to allow cleaner syntax and comments, and `toml`, with the default being `toml`. If you want to switch to `json5`, we can do that using a `Processor`!
+We'll need to apply this processor globally to our config, since the way we'll be changing the format is via the `Config.Builder` object a config class processor will provide.
+
+This processor needs to run before the config is read, so we're going to place it directly on the class:
+
+`src/main/com/example/example_mod/ExampleModConfig`:
+
+```java
+@Processor("processConfig")
+public class ExampleModConfig extends ReflectiveConfig {
+	public void processConfig(Config.Builder builder) {
+		// ...
+        builder.format("json5");
+	}
+
+	// ...
+}
+```
+
+With our knowledge of processors, this is simple! You can also use the config builder to add new fields, new sections, and update metadata (TODO LINK TO METADATA PAGE), on top of changing the format and using callbacks as we've already covered.
+
+## Adding multiple files
+
+For massive mods, a single config file, even organised into sections, can become unwieldy.
+Luckily, Quilt Config is designed to easily support adding multiple config files!
+To add a second config file, we must make another config class: let's call this one `ExampleModConfig2`.
+We'll also have to update the name of our original config file to be more specific:
+
+`src/main/com/example/example_mod/ExampleModConfig`:
+
+```java
+public class ExampleModConfig extends ReflectiveConfig {
+    public static final ExampleModConfig INSTANCE = QuiltConfig.create(ExampleMod.MOD_ID, "main", ExampleModConfig.class);
+}
+```
+
+Instead of using the mod ID as the name of our class, we call our original config `main` instead.
+Now let's create a second config:
+
+`src/main/com/example/example_mod/ExampleModConfig2`:
+
+```java
+public class ExampleModConfig2 extends ReflectiveConfig {
+    public static final ExampleModConfig2 INSTANCE = QuiltConfig.create(ExampleMod.MOD_ID, "secondary", ExampleModConfig2.class);
+}
+```
+
+Just the same thing as our original, but using the `ExampleModConfig2` class instead of `ExampleModConfig` everywhere. We also name it `secondary`, to go along with the `main` name of our original config.
+Now you can add whatever fields you'd like! With Quilt Config, you can repeat this process as much as you like, as long as no configs have duplicate names.
```

Comment:
Do all configs under a namespace go in a directory?
If so, worth mentioning, if not but qconf provides a way to do so, maybe worth explaining how?

Reply:
yes, all files go in a directory
I'll clarify!

Reply:
note that we metion this at the start of the getting started article, but I added a reminder here